packages/flutter/test/material/icon_button_test.dart

```diff
@@ -428,6 +428,33 @@ void main() {
     expect(focusNode.hasPrimaryFocus, isFalse);
   });
 
+  testWidgets('IconButton shows focus highlight.', (WidgetTester tester) async {
```

Comment:
Is this test related to this change?

Reply:
Whoops, how did this get in here?  No.  I'll move that to another PR.